terraform/vpc.tf

```diff
@@ -0,0 +1,47 @@
+module "vpc" {
+  // Keeping the 'registry.terraform.io/' prefix so Rider can find the module for autocompletion purposes
+  source  = "registry.terraform.io/terraform-aws-modules/vpc/aws"
+  version = "3.14.2"
+
+  name = join("-", ["gpg-application-vpc", var.env])
+  cidr = "10.0.0.0/16"
+
+  azs              = slice(data.aws_availability_zones.available.zone_ids, 0, 2)
+  public_subnets   = [cidrsubnet("10.0.0.0/16", 4, 0), cidrsubnet("10.0.0.0/16", 4, 1)]
+  database_subnets = [cidrsubnet("10.0.0.0/16", 4, 2), cidrsubnet("10.0.0.0/16", 4, 3)]
```

Comment:
This looks worryingly like our database subnets are a subset of our public subnets.

Reply:
database_subnets is a subset of subnets, and is private by default.